geode-wan/src/main/java/org/apache/geode/cache/client/internal/locator/wan/LocatorMembershipListenerImpl.java

```diff
@@ -170,6 +207,28 @@ private synchronized Object updateAllLocatorInfo(RemoteLocatorJoinRequest reques
     return new RemoteLocatorJoinResponse(this.getAllLocatorsInfo());
   }
 
+  private ExecutorService createThreadPool() {
```

Comment:
Are we worried about concurrency in this creation?

Reply:
As current implementation, start() method, is only invoked from InternalLocator constructor. If we are concerned about future use, I can make corrections.